src/vips/vobject.lua

```diff
@@ -77,13 +75,12 @@ local vobject_mt = {
 
         set = function(self, name, value)
             log.msg("vobject.set")
-            log.msg("  self =", self)
             log.msg("  name =", name)
             log.msg("  value =", value)
 
             local gtype = self:get_typeof(name)
             if gtype == 0 then
-                error(verror.get())
+                return false
```

Comment:
Same here, I think `set()` should throw an exception. Use `set_type()` for an exception-free one.